packages/cli/src/commands/destroy/scaffold/__tests__/scaffold.test.js

```diff
@@ -1,5 +1,21 @@
 global.__dirname = __dirname
+
+import fs from 'fs'
+
+import 'src/lib/test'
+import { getPaths, getDefaultArgs } from 'src/lib'
+
+import { defaults, files } from '../../../generate/scaffold/scaffold'
+import { tasks } from '../scaffold'
+
 jest.mock('fs')
+jest.mock('@babel/core', () => {
```

Comment:
babel is unhappy when you mock `fs`

Reply:
We're thinking of creating a wrapper around `fs` with a few common operations like write, read, unlink in `@redwoodjs/internal` that should be slightly more manageable to mock.